packages/abtest/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@woocommerce/abtest",
```

Comment:
It might be nice to call it `explat` rather than `abtest`, and we have plans to offer more than just a/b tests :)